modules/preloader/include/mistAPI.js

```diff
@@ -159,20 +157,23 @@ module.exports = () => {
                 const filteredId = prefix + filterId(id);
                 queue.push({ action: 'selectMenu', id: filteredId });
 
-                for (var e in this.entries) {
+                for (let e in this.entries) {
                     if ({}.hasOwnProperty.call(this.entries, e)) {
                         this.entries[e].selected = (e === filteredId);
```

Comment:
@frozeman actually this traverses through all entries, where `e === filteredId` returns either `true` or `false`, depending on the current element.